makkara/.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+    "env": {
+        "es2021": true,
+        "node": true
+    },
+    "extends": [
+        "airbnb-base"
+    ],
+    "parserOptions": {
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "rules": {
+        "no-console": "off"
```

Comment:
Should we use 4 space indents instead? How would everyone prefer? Imo it would make code more breathable and nicer to eyes.

```json
"rules": {
    "no-console": "off",
    "indent": [
        "error",
        4
    ]
}
```